### ISSUE TITLE
New files not visible in local mode

### ISSUE THREAD

**Issue by msample** [mjibson/esc#25]
The Open() function for the localFS (ie dev mode where original files are exposed) first checks that the file exists in the last esc-generated file list, and errors if it's not there.    More serious refactoring  & renaming of files would be permitted if this was simply a pass through to Open().    You wouldn't want this in production for security reasons though - perhaps it a new mode, DANGER_MODE.

**Comment by mjibson** [comment]
Yes, I'd like to do this. This is a backward-compatible API change, and I'd like to take the opportunity to add a real test suite when this option is done, too. So for now this will wait (even longer!) but it'd be real nice.
